os/kernel/log_dump/log_dump.c

```diff
@@ -109,10 +140,61 @@ int log_dump_init(void)
 	return LOG_DUMP_OK;
 }
 
-void log_dump_read_init(void)
+unsigned char *last_comp_block;
+size_t last_comp_block_size;
+int last_comp_block_ptr;
+bool last_block_compression;
+
+int log_dump_read_init (void)
+{
+	last_block_compression = true;
+	struct log_dump_msg_s msg;
+
+	msg.comp = true;
+	msg.ch = 'c';
+	mq_send(log_dump_get_mqfd(), (const char *)&msg, sizeof(struct log_dump_msg_s), 100);
+
+	while (last_block_compression);
+	return 0;
+}
+
+int log_dump_read_init_helper(void)
```

Comment:
What is the helper meaning?
It is not good to understand the purpose of this function.